src/scss/component/_container.scss

```diff
@@ -66,6 +56,12 @@
 	}
 }
 
+.registry__component--brand-tabs.o-tabs--big[role=tablist] {
+	@include oTypographyMargin($bottom: 5);
+	@include oTabsButtonTabsTheme($theme: 'mono', $selector: false);
```

Comment:
Not a huge deal but `@include oTabsButtonTabsTheme($theme: 'mono', $selector: false);` is repeated.